src/scss/grommet-core/_objects.content-card.scss

```diff
@@ -0,0 +1,159 @@
+$card-hover-color: #EBEBEB;
+$social-twitter-color: #1DA1F2;
+$social-facebook-color: #3B5998;
+$social-linkedin-color: #0077B5;
+
+@mixin social-icon-color($social-color) {
+  fill: $social-color;
+  stroke: $social-color;
+}
+
+.content-card {
+  @include media-query(palm) {
+    padding: 0;
+
+    &:not(:last-child) {
+      border-bottom: 1px solid $border-color;
+    }
+  }
+
+  &__social-icon {
+    .#{$grommet-namespace}control-icon-social-twitter {
+      @include social-icon-color($social-twitter-color);
+    }
+
+    .#{$grommet-namespace}control-icon-social-facebook {
+      @include social-icon-color($social-facebook-color);
+    }
+
+    .#{$grommet-namespace}control-icon-social-linkedin {
+      @include social-icon-color($social-linkedin-color);
+    }
+  }
+
+  .#{$grommet-namespace}control-icon-watch,
+  .#{$grommet-namespace}anchor:hover .#{$grommet-namespace}control-icon-watch {
+    stroke: none;
+    fill: $control-brand-color;
+  }
+
+  &__thumbnail {
+    width: 100%;
+    height: $inuit-base-spacing-unit * 11;
+    flex-shrink: 1;
+
+    .#{$grommet-namespace}anchor & {
+      @include media-query(lap-and-up) {
+        height: 100%;
+      }
+    }
+
+    .#{$grommet-namespace}control-icon-watch {
+      @include media-query(palm) {
+        // don't shrink video WatchIcon when in mobile
+        width: round($inuit-base-spacing-unit * 6);
+        height: round($inuit-base-spacing-unit * 6);
+      }
+    }
+  }
+
+  &__content {
+    // truncate description paragraph after 8 lines
+    .#{$grommet-namespace}paragraph {
+      overflow: hidden;
+      display: -webkit-box;
+      -webkit-line-clamp: 8;
+      -webkit-box-orient: vertical;
+      max-height: 176px; // 22 * 8 (p line-height * # lines)
```

Comment:
There's a lot of custom fixed pixel sizing in here. Can we do it in a way that is more flexible/relative? I'm just wondering of the content card is supposed to have a height, perhaps we should create a t-shirt sized height property on it and then set these sizes more proportional to that.